tests/installation/welcome.pm

```diff
@@ -149,6 +149,7 @@ sub run {
     ensure_fullscreen;
 
     assert_screen((is_sle('15+') && get_var('UPGRADE')) ? 'inst-welcome-no-product-list' : 'inst-welcome');
+    assert_screen('inst-welcome-no-product-list') if is_microos;
```

Comment:
```suggestion
    assert_screen((is_sle('15+') && get_var('UPGRADE') || is_microos) ? 'inst-welcome-no-product-list' : 'inst-welcome');
```

Reply:
I'd prefer a proper `if`/`else` at that point, it's not really readable anymore

Reply:
you mean like this?
```
    if (is_sle('15+') && get_var('UPGRADE')) || is_microos {
        assert_screen('inst-welcome-no-product-list');
    }
    else {
        assert_screen('inst-welcome');
    }
```

Reply:
Yes, just `    if (is_microos || (is_sle('15+') && get_var('UPGRADE'))) {`

Reply:
Though I wonder whether SUSE MicroOS shouldn't just have a `inst-welcome` needle like openSUSE MicroOS.